schedule/yast/yast_no_self_update/yast_no_self_update@s390x_kvm.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: yast_no_self_update
+description:    >
+  Test suite conducts installation with self-update explicitly disabled. No
+  hard checks are done that the self-updating is really disabled.
+  Installation is validated by successful boot and that YaST does not report
+  any issue.
+vars:
+  INSTALLER_NO_SELF_UPDATE: 1
+  YUI_REST_API: 1
+schedule:
+  product_selection:
+    - installation/product_selection/install_SLES
+    - installation/validate_no_self_update
+  installation:
```

Comment:
The whole `installation:` section is now a copy of what is already in the default. 
Better would be to have:

```
    default_systemd_target :
    - installation/installation_settings/validate_default_target
```

here and completely delete the `installation:` section because it is already in the default file, of course also returning to the empty hook in the default file as mentioned above.

Reply:
Modifiled.